### PR TITLE
added a separate section - supported DB versions

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/wavemaker-release-notes/v10-7-0.md [hunk 1]
```diff
@@ -42,20 +44,21 @@ sidebar_label: "Release v10.7.0"
 
 ### Query Optimization for Faster DB Imports
 
-- Metadata queries invoked to fetch tables' constraints information are optimized for ~600x faster database imports in an application.
+- Metadata queries invoked to fetch tables' constraints information are optimized with ~600x faster database imports in an application.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-7-0.md:47]: Importing Databases into WaveMaker application is now much faster. We have optimised the queries that were used to read the metadata of the databases.

File: learn/wavemaker-release-notes/v10-7-0.md [hunk 2]
```diff
@@ -65,6 +68,14 @@ Introduced the following methods to get default options provided for the Richtex
 - `getLib()`: Returns the library used for the widget.
 - `overrideDefaults(options)`: Customize the options which the library supports.
 
+### Backend Enhancements
+
+- For Rest/Swagger services, the rest call goes from the backend when you enable proxy, making the direct path empty in service definitions.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-7-0.md:73]: i didn't understand this. Does not pass the "so, what" test

File: learn/wavemaker-release-notes/v10-7-0.md [hunk 3]
```diff
@@ -65,6 +68,14 @@ Introduced the following methods to get default options provided for the Richtex
...
+- Introduced the Allowed host property in the security tab of the profile configuration. It prevents host header injections and lets application developers control the allowed hosts of the application.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-7-0.md:74]: Prevents HTTP Host header injection attacks. Developers can specify the host names where the WaveMaker application is deployed and platform will rejects HTTP requests which have different values in Host header than this list.

File: learn/wavemaker-release-notes/v10-7-0.md [hunk 4]
```diff
@@ -65,6 +68,14 @@ Introduced the following methods to get default options provided for the Richtex
...
+- Added support in the Deployed Apps portal to enable **Creation** and **Deployment** of application in 1.18 and 1.19 Kubernetes clusters in Digital Ocean.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-7-0.md:75]: Ability to deploy apps to 1.18 and 1.19 Kubernetes clusters in Digital Ocean

File: learn/wavemaker-release-notes/v10-7-0.md [hunk 5]
```diff
@@ -65,6 +68,14 @@ Introduced the following methods to get default options provided for the Richtex
...
+- Introduced the **Max Authentication Age** property in the project security configuration. It permits developers to allow users to stay authenticated for the specified duration when the identity provider (idP) allows.
+- Introduced the `Max Sessions` property under the "Login configuration" section. With this, a developer can configure the duration for a logged-in user's session's life.
```
Comment (prasanth-reddy-wavemaker-com) [inline learn/wavemaker-release-notes/v10-7-0.md:77]: Incorrect. Configure number of concurrent sessions your application users can have by setting the property `Max Sessions`. This settings can be found in the Login Configuration tab of the Security menu.

**Review by prasanth-reddy-wavemaker-com — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-7-0.md (hunk 2 above)
Comment (swethakundaram) [reply learn/wavemaker-release-notes/v10-7-0.md:73]: @prakashbuthukuri, do you know what problem it solves?

Comment (prakashbuthukuri) [reply learn/wavemaker-release-notes/v10-7-0.md:75]: If we didnt make the direct path as empty, then the full service path is visible in the network/developer console
(full path -> host name, request path and any path or query parameters configured for that service). 
Some customers raised security concern over exposing the direct path when proxy is enabled. 
So to address that we make the direct path as empty when proxy is enabled.

Comment (prasanth-reddy-wavemaker-com) [reply learn/wavemaker-release-notes/v10-7-0.md:75]: Lets remove this line @swethakundaram

File: learn/wavemaker-release-notes/v10-7-0.md (hunk 1 above)
Comment (prasanth-reddy-wavemaker-com) [reply learn/wavemaker-release-notes/v10-7-0.md]: can you rephrase this sentence like i suggested above. We should explain what the impact of the improvement is @swethakundaram

**Review by prasanth-reddy-wavemaker-com — APPROVED** [review]
